icons/jestjs/jest-line.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The line version looks like the `jest-plain.svg` file but with a different color.